sdk/keyvault/keyvault-certificates/README.md

```diff
@@ -162,7 +163,7 @@ const url = `https://${vaultName}.vault.azure.net`;
 
 // Change the Azure Key Vault service API version being used via the `serviceVersion` option
 const client = new CertificateClient(url, credential, {
-  serviceVersion: "7.0"
+  serviceVersion: "7.0",
```

Comment:
I know TS likes this, but I thought this was a violation in JS, or is that just older language servers?

Reply:
ES5 restored sanity by allowing it in most places 😄. I _believe_ prior to that only arrays were allowed trailing commas, but I don't remember exactly. Anywho, we recently standardized across our repository on trailing commas and prettier helped by adding them on save here.